packages/backend/src/apps/m365-excel/actions/get-table-rows/get-data-out-metadata.ts

```diff
@@ -0,0 +1,62 @@
+import type { IDataOutMetadata, IExecutionStep } from '@plumber/types'
+
+import { dataOutSchema } from './schemas'
+
+async function getDataOutMetadata(
+  executionStep: IExecutionStep,
+): Promise<IDataOutMetadata> {
+  const { dataOut: rawDataOut } = executionStep
+  if (!rawDataOut) {
+    return null
+  }
+
+  // Parse the data first to get proper type information
+  const dataOut = dataOutSchema.parse(rawDataOut)
+
+  const metadata: IDataOutMetadata = {
+    rowsFound: {
+      label: 'Number of rows found',
+      order: 2,
+    },
+  }
+
+  if (dataOut.rowsFound === 0) {
+    return metadata
+  }
```

Comment:
can i check: is there a specific reason why list of rows found won't be returned for excel for 0 rows but for tiles it is being returned?